src/initialization.js

```diff
@@ -0,0 +1,286 @@
+const Apify = require('apify');
+const _ = require('lodash');
+const HeaderGenerator = require('header-generator');
+const { TYPES, LABELS } = require('./constants');
+
+const fns = require('./functions');
+
+const { utils: { log } } = Apify;
+
+const {
+    getUrlData,
+    extendFunction,
+    isOverItems,
+    minMaxDates,
+    // eslint-disable-next-line no-unused-vars
+    createGetSimpleResult,
+} = fns;
+
+/**
+ * Throws error if the provided input is invalid.
+ * @param {{ search: String, startUrls: any[], zpids: any[] }} input
+ */
+const validateInput = (input) => {
+    if (!(input.search && input.search.trim().length > 0) && !input.startUrls && !input.zpids) {
+        throw new Error('Either "search", "startUrls" or "zpids" attribute has to be set!');
+    }
+};
+
+/**
+ *
+ * @param {{ minDate: string, maxDate: string }} input
+ * @returns minMaxDate
+ */
+const initializeMinMaxDate = ({ minDate, maxDate }) => {
+    return minMaxDates({
```

Comment:
I would probably use class instead of this function. It also feels like it is a wrapping function with just renamed parameters. I would not do function aliases in this relatively small codebase.